src/plantumlv2/pu_manager.py

```diff
@@ -32,9 +31,7 @@ def render_pu(graph: BTGraph, config: dict):
         _save_plantuml_str(save_location, plant_uml_str)
 
 
-def render_diff_pu(
-    local_bt_graph: BTGraph, remote_bt_graph: BTGraph, config: dict
-):
+def render_diff_pu(local_bt_graph: BTGraph, remote_bt_graph: BTGraph, config: dict):
```

Comment:
so this is only formatting as far as I can tell? 
should we settle on a common formatter for the project? Otherwise we will always have diffs that just change the formatting and they just slow reviewing down?
I hear people like black, so I would go with that.

Reply:
Yes, it is probably a good idea to settle on a common formatted for the project, and also for all of us to agree on the Python style that we use in this project. We don't have any specific preferences for the formatter that we use, so we could give Black a try.